acceptance_tests/features/reply_to_message_internal.feature

```diff
@@ -40,3 +40,27 @@ Feature: Reply to message internal
     When they view the message
     And they reply to the message
     Then they receive confirmation that the message has been sent
+
+  @sm123_s01
+  Scenario: User is able to close a conversation
```

Comment:
Should Closed conversation tests be in their own feature file? doesn't seem right to be in `reply_to_message_internet.feature`

Reply:
I put them in there because the opening and closing of conversations happens from the reply page.  Agreed to keep them where they are for now and to raise a card to look at the secure message tests in general.